src/components/Avatar/Avatar.tsx

```diff
@@ -1,28 +1,41 @@
-// @ts-check
 import React, { useEffect, useState } from 'react';
-import PropTypes from 'prop-types';
+
+export type AvatarProps = {
+  /** image url */
```

Comment:
I believe @ambaldwin updated the text here to be a little more semantic in the context of our documentation, we're prob gonna need to update a lot of the old prop type text to make more sense from a user standpoint

Reply:
Yeah, I currently have the below for the props text for Avatar:

/** Image URL or default is an image of the first initial of the name if there is one  */
  image?: string;
  /** Name of the image, used for title tag fallback */
  name?: string;
  /** click event handler */
  onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
  /** mouseOver event handler */
  onMouseOver?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
  /** Shape of the avatar - circle, rounded or square, and the default is circle */
  shape?: AvatarShape;
  /** Size in pixels and the default is 32px */
  size?: number;